src/docs/product/accounts/pricing.mdx

```diff
@@ -103,27 +74,71 @@ We have three tables for [errors](#errors-pricing), [transactions](#transactions
 
 <!-- prettier-ignore-end -->
 
-You can buy prepaid volume in increments as shown in the table below:
-
 <!-- prettier-ignore-start -->
 
 <!-- Don't remove spaces from first column header or number ranges in column will break onto two lines making them hard to read -->
 
-| Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |
-|---|---|---|---|---|---|
-| >50k-100k | 50k | >100k-250k | 150k | >25 - 100 (GB) | 25 (GB) |
-| >100k-500k | 100k | >250k-500k | 250k | >100 - 10k (GB) | 100 (GB) |
-| >500k-2M | 500k | >500k-10M | 500k |  |  |
-| >2M-20M | 1M | >10M-100M | 5M |  |  |
-| >20M-50M | 2M |  |  |  |  |
+#### Consolidated Pricing
+
+| Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | 
+|---|---|---|---|
+| >50k-100k | 50k | >100k-250k | 150k | 
+| >100k-500k | 100k | >250k-500k | 250k | 
+| >500k-2M | 500k | >500k-10M | 500k |
+| >2M-20M | 1M | >10M-100M | 5M | 
+| >20M-50M | 2M |  |  |
+
+| Attachments Volume | Purchase Increment | Replay Volume | Purchase Increment |
+|---|---|---|---|
+| >25 - 100 (GB) | 25 (GB) | >10k - 25k | 15k |
+| >100 - 10k (GB) | 100 (GB) | >25k - 100k | 25k |
+|  |  | >100k - 1M | 100k |
 
 <!-- prettier-ignore-end -->
 
-On-demand purchasing isn't restricted by any increments since you set a dollar amount for your budget, rather than a certain amount of data. So if you want to purchase an amount of prepaid volume data that falls between increments, you should set that up as on-demand budget.
+If the amount of prepaid data you want to buy falls in-between increments, you may want to consider setting an on-demand budget, (which is based on a dollar amount rather than data volume), instead. 
+
```

Comment:
```suggestion
If the amount of prepaid data you want to buy falls in-between increments, you may want to consider setting an on-demand budget instead. Because on-demand is set in dollar amounts rather than data volume.

```